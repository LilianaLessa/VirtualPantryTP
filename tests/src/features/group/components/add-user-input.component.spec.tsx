import * as React from "react";
import * as renderer from "react-test-renderer";
import AddUserInput from "../../../../../src/features/group/components/add-user-input.component";

describe("App", () => {
  it("snapshot test", () => {
    const tree = renderer
      .create(<AddUserInput addUserCallback={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
