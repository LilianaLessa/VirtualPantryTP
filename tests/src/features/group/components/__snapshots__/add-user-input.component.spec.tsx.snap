// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View
  className="sc-beqWaB knyjdu"
>
  <View
    style={
      {
        "width": "100%",
      }
    }
  >
    <View
      pointerEvents="none"
      style={
        [
          {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 6,
          },
          {
            "backgroundColor": "rgba(255, 251, 254, 1)",
            "borderColor": "rgba(121, 116, 126, 1)",
            "borderRadius": 4,
            "borderWidth": 1,
          },
          undefined,
        ]
      }
      testID="text-input-outline"
    />
    <View>
      <View
        style={
          [
            {
              "paddingBottom": 0,
            },
            {
              "minHeight": 56,
              "paddingTop": 8,
            },
          ]
        }
      >
        <View
          collapsable={false}
          pointerEvents="none"
          style={
            {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "transform": [
                {
                  "translateX": 0,
                },
              ],
              "zIndex": 3,
            }
          }
        >
          <Text
            collapsable={false}
            maxFontSizeMultiplier={1.5}
            numberOfLines={1}
            style={
              {
                "backgroundColor": "rgba(255, 251, 254, 1)",
                "color": "transparent",
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": undefined,
                "left": 8,
                "letterSpacing": 0.15,
                "lineHeight": undefined,
                "opacity": -0.6666666666666667,
                "paddingHorizontal": 0,
                "position": "absolute",
                "textAlign": "left",
                "top": 33,
                "transform": [
                  {
                    "translateX": 0,
                  },
                  {
                    "translateY": 0,
                  },
                  {
                    "scale": 1,
                  },
                  {
                    "scaleY": 1,
                  },
                ],
                "width": -16,
                "writingDirection": "ltr",
              }
            }
          >
            Add user
          </Text>
          <Text
            collapsable={false}
            maxFontSizeMultiplier={1.5}
            numberOfLines={1}
            onLayout={[Function]}
            style={
              {
                "color": "rgba(103, 80, 164, 1)",
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": undefined,
                "left": 0,
                "letterSpacing": 0.15,
                "lineHeight": undefined,
                "opacity": 0,
                "paddingHorizontal": 16,
                "position": "absolute",
                "textAlign": "left",
                "top": 32,
                "transform": [
                  {
                    "translateX": 0,
                  },
                  {
                    "translateY": 0,
                  },
                  {
                    "scale": 1,
                  },
                ],
                "writingDirection": "ltr",
              }
            }
            testID="text-input-outlined-label-active"
          >
            Add user
          </Text>
          <Text
            collapsable={false}
            maxFontSizeMultiplier={1.5}
            numberOfLines={1}
            style={
              {
                "color": "rgba(73, 69, 79, 1)",
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": undefined,
                "left": 0,
                "letterSpacing": 0.15,
                "lineHeight": undefined,
                "opacity": 0,
                "paddingHorizontal": 16,
                "position": "absolute",
                "textAlign": "left",
                "top": 32,
                "transform": [
                  {
                    "translateX": 0,
                  },
                  {
                    "translateY": 0,
                  },
                  {
                    "scale": 1,
                  },
                ],
                "writingDirection": "ltr",
              }
            }
            testID="text-input-outlined-label-inactive"
          >
            Add user
          </Text>
        </View>
        <TextInput
          className="sc-jTrPJq INhBu"
          editable={true}
          keyboardType="email-address"
          maxFontSizeMultiplier={1.5}
          multiline={false}
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          placeholder=" "
          placeholderTextColor="rgba(73, 69, 79, 1)"
          selectionColor="rgba(103, 80, 164, 1)"
          style={
            [
              {
                "margin": 0,
                "zIndex": 1,
              },
              {
                "height": 48,
              },
              {
                "paddingBottom": 0,
                "paddingTop": 0,
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": undefined,
                "letterSpacing": 0.15,
                "lineHeight": undefined,
                "paddingHorizontal": 16,
                "textAlign": "left",
                "textAlignVertical": "center",
              },
              false,
              [
                {},
              ],
              undefined,
            ]
          }
          testID="text-input-outlined"
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
</View>
`;
